app/config/config.yml

```diff
@@ -2,8 +2,108 @@ imports:
     - { resource: parameters.yml }
```

Comment:
Misschien inenting dan ook gelijktrekken. Want hier is 4 spaces, en bij de rest maar 2